.rubocop_todo.yml

```diff
@@ -79,6 +79,7 @@ RSpec/ContextWording:
     - 'spec/models/labware_creators/common_file_handling/csv_file_for_tube_rack_spec.rb'
     - 'spec/models/labware_creators/common_file_handling/csv_file_for_tube_rack_with_rack_barcode_spec.rb'
     - 'spec/models/labware_creators/concentration_binned_plate_spec.rb'
+    - 'spec/models/labware_creators/concentration_binned_full_plate_spec.rb'
```

Comment:
strictly we should try to reduce these todo's not add more. although adding disables, which tends to be what i do, is not that much better.